### PR TITLE
Feesheet code type strike out bug fix.

### PR DESCRIPTION
**Pull request by sjpadgett**
- ignore adding justify dialog anchor if procedure is being deleted.
- refactor deprecated `<strike>` with `<del>` style
- added spinner for justify dialog fetch
- some code clean up

### CODE REVIEW

**Comment by sjpadgett** [comment]
See: https://community.open-emr.org/t/multiple-fee-sheet-improvements/8017/34
will port this back to v5.0.2 as soon as this is merged.

File: interface/forms/fee_sheet/review/js/fee_sheet_core.js [hunk 1]
```diff
@@ -136,7 +138,7 @@ function justify_start(evt) {
         }
     }).done(function (data) {
         // now init justify
-        let parent = jqElem.parent()
+        let parent = jqElem.parent();
```
Comment (stephenwaite) [inline interface/forms/fee_sheet/review/js/fee_sheet_core.js]: guess we should've noticed this in the disappearing modifier :1st_place_medal:

Comment (sjpadgett) [reply interface/forms/fee_sheet/review/js/fee_sheet_core.js]: yep, I have a bad habit of missing the small stuff too.  I rely to much on cs fixing for me but I don't have turned on for JS.

Comment (stephenwaite) [reply interface/forms/fee_sheet/review/js/fee_sheet_core.js]: neither does the project obviously, if managed to update codesniff that would be quite the undertaking :plate_with_cutlery:

Comment (sjpadgett) [reply interface/forms/fee_sheet/review/js/fee_sheet_core.js]: i've been tempted to format them if I touch one but then it gets confusing for reviewer. Best to do all at once one day like was done for php scripts in the beginning...

Comment (stephenwaite) [reply interface/forms/fee_sheet/review/js/fee_sheet_core.js]: or this one https://github.com/openemr/openemr/pull/2527/commits/118a2de66a0d8e732455e0dcebf28e845661ac51 :)

**Pull request merged**